frontend/server/k8s-helper.ts

```diff
@@ -168,6 +162,9 @@ export async function getTensorboardInstance(
  */
 
 export async function deleteTensorboardInstance(logdir: string): Promise<void> {
+  if (!namespace) {
+    throw new Error(`Cannot get namespace from ${namespaceFilePath}.`);
+  }
   if (!k8sV1CustomObjectClient) {
```

Comment:
In other places, the checks on custom object client are removed, can we also remove the check here?

Reply:
Thanks! Yes, we can also remove this

Reply:
Fixed